src/main/scala/is/hail/utils/Py4jUtils.scala

```diff
@@ -0,0 +1,43 @@
+package is.hail.utils
+
+import scala.collection.JavaConverters._
+
+trait Py4jUtils {
+  def arrayToArrayList[T](arr: Array[T]): java.util.ArrayList[T] = {
+    val list = new java.util.ArrayList[T]()
+    for (elem <- arr)
+      list.add(elem)
+    list
+  }
+
+  def iterableToArrayList[T](it: Iterable[T]): java.util.ArrayList[T] = {
+    val list = new java.util.ArrayList[T]()
+    for (elem <- it)
+      list.add(elem)
+    list
+  }
+
+  def arrayListToISeq[T](al: java.util.ArrayList[T]): IndexedSeq[T] = al.asScala.toIndexedSeq
```

Comment:
That requires a classtag, and when I asked for the classtag I got a couple of other odd compiler errors.  I'll leave it as indexed seq for now with a FIXME note